larray/core/array.py

```diff
@@ -6019,7 +6019,9 @@ def to_hdf(self, filepath, key, *args, **kwargs):
         >>> a.to_hdf('test.h5', 'a')  # doctest: +SKIP
         """
         key = _translate_key_hdf(key)
-        self.to_frame().to_hdf(filepath, key, *args, **kwargs)
+        with LHDFStore(filepath) as store:
+            store.put(key, self.to_frame())
+            store.get_storer(key).attrs.type = 'LArray'
```

Comment:
We might want to call this 'Array' instead if we decide to implement #611 later.